spec/api-menu-spec.js

```diff
@@ -860,11 +860,15 @@ describe('Menu module', () => {
           submenu: [
             {
               label: 'Test Item',
-              accelerator: 'Ctrl+T',
-              click: () => {
-                // Test will succeed, only when the menu accelerator action
-                // is triggered
-                Promise.resolve()
+              accelerator: 'T',
+              click: (a, b, event) => {
+                expect(event).to.deep.equal({
```

Comment:
I don't think this click handler is actually getting tested?

E.G. If I change line 870 to `triggeredByAccelerator: false`, this test still passes.

Reply:
Weird, the test seems to be working as intended on my end! See screen recording below (hope it's readable 😅):
![screencast 2019-06-18 15-13-27](https://user-images.githubusercontent.com/16010076/59723661-389a9c00-91dc-11e9-9410-dd0f2f89d8dc.gif)

Reply:
Ah I see what's happening. I tested in `--ci` mode which for some reason is prevents the click handler from being reached. Interesting, but not this PR's issue.